docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+services:
+    composer:
+        build:
+            context: .
+            dockerfile: Dockerfile.composer
+            args:
+                composer_dev_arg: ${COMPOSER_DEV_ARG}
+        image: libero/browser_composer:${IMAGE_TAG}
+        command: 'true'
```

Comment:
Never found a way to have `docker-compose` services that don't actually run.

Reply:
`/bin/true` could be immediately understood